python/interpret-core/interpret/glassbox/_ebm/_ebm.py

```diff
@@ -905,19 +706,19 @@ def fit(self, X, y, sample_weight=None, bags=None, init_score=None):  # noqa: C9
                     bag,
                     init_score_local,
                     term_features,
-                    inner_bags,
+                    self.inner_bags,
                     term_boost_flags,
                     self.learning_rate,
-                    min_samples_leaf,
-                    min_hessian,
+                    self.min_samples_leaf,
+                    self.min_hessian,
                     self.max_leaves,
-                    greediness,
-                    cyclic_progress,
-                    smoothing_rounds,
+                    self.greediness,
+                    self.cyclic_progress,
+                    self.smoothing_rounds,
                     nominal_smoothing,
                     self.max_rounds,
                     early_stopping_rounds_local,
-                    early_stopping_tolerance,
+                    self.early_stopping_tolerance,
```

Comment:
The attributes that didn't originally have self. should not exist on the DPEBMModel objects.